dcpy/test/metadata/test_metadata.py

```diff
@@ -116,8 +143,106 @@ def test_socrata_column_overrides_wkb_geom():
 
 
 def test_validating_valid_data():
-    dataset = metadata.dataset_package.get_dataset("primary_shapefile")
-    fake_ds = _generate_fake_dataset(100, columns=dataset.get_columns(metadata))
+    dataset = metadata.dataset_package.get_dataset("primary_csv")
+    fake_ds = generate_fake_dataset(100, columns=dataset.get_columns(metadata))
     results = validate.validate_df(fake_ds, dataset, metadata)
 
     assert results == [], "No Errors should have been found in source data"
+
+
+def test_invalid_standardized_values():
+    dataset = metadata.dataset_package.get_dataset("primary_csv")
+    ROW_COUNT = 100
+    fake_ds = generate_fake_dataset(ROW_COUNT, columns=dataset.get_columns(metadata))
+
+    INVALID_OWNERSHIP_VALUES = ["X", "Y", "Z"]
+    fake_ds.loc[0, "non_nullable_ownership"] = INVALID_OWNERSHIP_VALUES[0]
+    fake_ds.loc[1, "non_nullable_ownership"] = INVALID_OWNERSHIP_VALUES[1]
+    fake_ds.loc[2:, "non_nullable_ownership"] = INVALID_OWNERSHIP_VALUES[2]
+
+    results = validate.validate_df(fake_ds, dataset, metadata)
+    assert (
+        len(results) == 1
+    ), "One error should have been found for invalid standardized values"
+
+    # Assert that the error message should mention the invalid values with their counts
+    result_msg = results[0][1]
+    assert (
+        f"'{INVALID_OWNERSHIP_VALUES[0]}': 1," in result_msg
+    ), "The error message should include the invalid value and count"
+    assert (
+        f"'{INVALID_OWNERSHIP_VALUES[1]}': 1," in result_msg
+    ), "The error message should include the invalid value and count"
+    assert (
+        f"'{INVALID_OWNERSHIP_VALUES[2]}': {ROW_COUNT - 2}" in result_msg
```

Comment:
there are 98 rows with a value of `Z`? looked to me like there'd only be one row with it because of `fake_ds.loc[2:, "non_nullable_ownership"] = INVALID_OWNERSHIP_VALUES[2]`

Reply:
yeah, it's the `2:` that means basically 2:99

Reply:
haha sorry I didn't notice the `:`